docs/metrics.asciidoc

```diff
@@ -397,16 +440,16 @@ MeterRegistry registry = new SimpleMeterRegistry(new SimpleConfig() {
             return null;
         }
     }, Clock.SYSTEM);
-
-```
+----
 
 When using Spring Boot, you can use the `management.metrics.export.simple` prefix to configure via `application.properties`
```

Comment:
I didn't see the Spring Boot section when I used these docs to setup my micrometer integration.
Did we consider solutions to make it easy for developers using Spring Boot to find our dedicated doc. As Spring Boot is very popular, we may even consider moving this section before the "Do It Yourself" section.

Reply:
Let's add a separate section for Spring Boot and add a table of contents to the top of the page.

```suggestion
==== Get started with Spring Boot

The easiest way to get started with Spring Boot is to add a dependency to [Spring Boot Actuator](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html) if you haven't already. Spring Boot Actuator provides dependency management and auto-configuration for Micrometer.

Use the `management.metrics.export.simple` prefix to configure via `application.properties`
```